crates/core/src/network/reducers/mod.rs

```diff
@@ -69,6 +71,7 @@ fn resolve_reducer(action_wrapper: &ActionWrapper) -> Option<NetworkReduceFn> {
         Action::Query(_) => Some(reduce_query),
         Action::QueryTimeout(_) => Some(reduce_query_timeout),
         Action::GetValidationPackage(_) => Some(reduce_get_validation_package),
+        Action::GetValidationPackageTimeout(_) => Some(reduce_get_validation_package_timeout),
```

Comment:
There is a question of Is there anyway we can combine QueryTimeout and GetValidationPackageTimeout? I would imagine the code is similar? I imagine just Action::Timeout and maybe with QueryTimeout and :GetValidationPackageTimeout as variants?